inference-engine/include/ie/gpu/gpu_context_api_ocl.hpp

```diff
@@ -189,15 +205,13 @@ static inline Blob::Ptr make_shared_blob_nv12(RemoteContext::Ptr ctx, cl::Image2
     size_t height = nv12_image_plane_y.getImageInfo<CL_IMAGE_HEIGHT>();
 
     // despite of layout, blob dimensions always follow in N,C,H,W order
-    TensorDesc ydesc(Precision::U8, { 1, 1, height, width }, Layout::NHWC);
+    TensorDesc ydesc(Precision::U8, {1, 1, height, width}, Layout::NHWC);
 
-    ParamMap blobParams = {
-        { GPU_PARAM_KEY(SHARED_MEM_TYPE), GPU_PARAM_VALUE(OCL_IMAGE2D) },
-        { GPU_PARAM_KEY(MEM_HANDLE), static_cast<gpu_handle_param>(nv12_image_plane_y.get()) }
-    };
+    ParamMap blobParams = {{GPU_PARAM_KEY(SHARED_MEM_TYPE), GPU_PARAM_VALUE(OCL_IMAGE2D)},
+                           {GPU_PARAM_KEY(MEM_HANDLE), static_cast<gpu_handle_param>(nv12_image_plane_y.get())}};
     Blob::Ptr y_blob = std::dynamic_pointer_cast<Blob>(casted->CreateBlob(ydesc, blobParams));
 
-    TensorDesc uvdesc(Precision::U8, { 1, 2, height / 2, width / 2 }, Layout::NHWC);
+    TensorDesc uvdesc(Precision::U8, {1, 2, height / 2, width / 2}, Layout::NHWC);
```

Comment:
Can keep as is?

Reply:
I can. Clang-format has an option `Cpp11BracedListStyle: false` to do such things, but IMHO, we have both styles in the IE and I don't know which style applied more widely